com.microsoft.mrtk.graphicstools.unity/Samples~/Clipping Primitives/Scripts/Rotate.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Rotate : MonoBehaviour
```

Comment:
Could you put this class in the `Microsoft.MixedReality.GraphicsTools.Samples.ClippingPrimitives` namespace?